tachyon/zk/plonk/halo2/synthesizer.h

```diff
@@ -89,14 +89,30 @@ class Synthesizer {
     }
   }
 
-  // Return |challenge_| as a vector.
+  // Return |challenge_| as a vector instead of |btree_map|.
```

Comment:
This doesn't seem to be related to the commit title "feat(zk): add takers of Synthesizer"